src/com/complexible/common/csv/CSV2RDF.java

```diff
@@ -174,7 +174,7 @@ private String insertPlaceholders(List<String> cols, File templateFile) throws E
 			return sb.toString();
 		}
 
-		private ValueProvider valueProviderFor(String varName, List<String> cols) throws Exception {
+		private ValueProvider valueProviderFor(String varName, List<String> cols) throws CreateValueProviderException {
```

Comment:
Remove the declaration of thrown exception 'com.complexible.common.csv.CreateValueProviderException', as it cannot be thrown from method's body.